imsv-docs-docusaurus/openapi/endpoints/kyc/get-kyc-profile.yaml

```diff
@@ -5,11 +5,19 @@ description: Get a KYC profile by account id if it exists.
 parameters:
   - name: accountId
     in: path
-    description: Id of the account.
+    description: Id of the cardholder or identity account.
     example: 225d85e65495722bf6517ea0ba0d6f56
     required: true
     schema:
       type: string
+  - name: includePersonalDetails
+    in: query
+    description: Include personal details (name, date of birth, etc) in the response. Requires additional permissions.
```

Comment:
should we clarify what those additional permissions are? is it the ones you get with an admin key or something else?

Reply:
there is a gap with permission documentation, so it is impossible to reference a specific permission. In this case, it is `kyc-profile:read-personal-details`, and just mentioning would be useless without mentioning what roles have that permission. Mentioning roles will mean a need to document roles as well, so I am reluctant to do it

Reply:
I see. It's just the current description is confusing without whose missing bits. Maybe there's a way to rephrase it?

Reply:
I could omit "Requires additional permissions."

Reply:
I could omit "Requires additional permissions."

Reply:
Hmm but that might also raise some questions like "why does this property not work" 😂 I'm not sure what's the best way around it tbh. Any other opinions?
I'm happy with it otherwise, so maybe we could address it in a follow-up PR